### PR TITLE
Feat/3604  add auto invite plugin

### PR DESCRIPTION
**Pull request by inkhey**
related to #3604 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: backend/official_plugins/tracim_backend_autoinvite/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Auto-Invite Plugin
+
+This internal plugin:
+- add all new user to open space
+- add all user to new created open space
```
Comment (grignards) [inline backend/official_plugins/tracim_backend_autoinvite/README.md:5]:
```suggestion
- adds every new user to all open spaces
- adds all users to every created open space
```

File: backend/official_plugins/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Officials Backend Plugins
+
+This dir contain all officials backend plugins
+These are "internal" (using internal and "unstable" API) plugins.
+You can active them by moving the plugins you need in `plugin.folder_path` dir.
```
Comment (grignards) [inline backend/official_plugins/README.md:5]:
```suggestion
This directory contains all official backend plugins
Those plugins are implemented using an experimental API that is planned to change: if you develop your own plugin using this API, expect it to break without notice.
You can enable them by moving the plugins you need in the `plugin.folder_path` directory.
```

File: backend/official_plugins/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+## Officials Backend Plugins
```
Comment (grignards) [inline backend/official_plugins/README.md:1]:
```suggestion
## Official Backend Plugins
```

**Review by grignards — CHANGES_REQUESTED** [review]
Only some grammar/typo on the documentation.

**Comment by grignards** [comment]
And don't forget to use asf-algoo for this PR.

**Review by grignards — APPROVED** [review]

**Review by raphj — APPROVED** [review]

**Comment by asf-algoo** [comment]
moved to https://github.com/tracim/tracim/pull/3730

**Pull request closed without merging**